### DISCUSSION TITLE
Welcome to cbeta-api Discussions!

### DISCUSSION THREAD

**Discussion by RayCHOU** [DILA-edu/cbeta-api#13] [General]
## 👋 Welcome!
GitHub 新增了 Discussion 功能，現在不必開 Issues 就可以討論了。
您可以在 [這裡 Discussions](https://github.com/DILA-edu/cbeta-api/discussions) ：
  * 提出問題
  * 分享想法
  * 與其他社區成員互動。

**Comment by MrMYHuang** [comment]
請問可能有法鼓全集的API嗎？因為有位大德建議我可以開發法鼓全集的app。
我認為技術上問題應該不大，我的電子佛典app程式碼應該只須作小修改就可以改出來。但我有稍微了解線上版法鼓全集的版權宣告：https://ddc.shengyen.org/pc.htm
有較多的使用限制。雖然知道可能性不高，但我還是想知道有無可能開發另一款法鼓全集app？阿彌陀佛

**Reply by RayCHOU** [reply]
隨喜您的發心，
我會將您的詢問轉給權責單位。

**Reply by MrRogerHuang** [reply]
謝謝您。

**Comment by RayCHOU** [comment]
對了，[Discussions](https://github.com/DILA-edu/cbeta-api/discussions) 那邊有幾個簡單的討論區分類，
可以在那邊開新討論串。
